### DISCUSSION TITLE
[Added] Add Aria2 fast command line downloader

### DISCUSSION THREAD

**Discussion by devilschile2** [retronas/discussions#51] [Ideas]
Adding Aria2, the fast command line downloader in the distribution, can provide a great tool for downloading content directly to retronas using urls from a file htat later can be imported using the tools already available in Retronas.

Aria2 supports HTTP/HTTPS, FTP, SFTP, BitTorrent and Metalink files.

**Comment by sairuk** [comment]
i honestly thought we had this tool listed on the ideas page already but when looking this morning it seems not.

Thanks for the suggestion, i've added it to the ideas page

**Reply by devilschile2** [reply]
I found it strange that it was not, and pitched the idea in case you found it useful. 

Congrats for the project btw. I already was using Nas storage before, but when I tried Retronas I found tha it made my live much easier in every way.

Thank you so much!

**Reply by sairuk** [reply]
aria2 has now been added into the testing branch

https://github.com/danmons/retronas/wiki/aria2

**Reply by devilschile2** [reply]
Awesome!!  
Thank you again!
